src/win/win-backend.cpp

```diff
@@ -15,6 +15,7 @@
 #include <chrono>
 #include <Windows.h>
 #include <dbt.h>
+#include <cctype> // std::tolower
```

Comment:
tolower is not a part of the PR. Where is it used?

Reply:
Used by [line 248](https://github.com/icarpis/librealsense/blob/bdbb42e66256a8920d9cfbae666c9019e4f36f13/src/win/win-backend.cpp#L248)